api/v1alpha1/podhttpchaos_types.go

```diff
@@ -181,6 +186,26 @@ const (
 	PodHttpResponse PodHttpChaosTarget = "Response"
 )
 
+// PodHttpChaosTLS contains the tls config for HTTPChaos
+type PodHttpChaosTLS struct {
+	// SecretName represents the name of required secret resource
+	SecretName string `json:"secretName,omitempty"`
+
+	// SecretNamespace represents the namespace of required secret resource,
+	// should be the same with deployment/chaos-controller-manager in most cases
```

Comment:
> should be the same with deployment/chaos-controller-manager in most cases

please remove this comment 🤣 there is no reason to make that suggestion/prediction